cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -575,6 +575,11 @@ func runClusterCreate(scConfig *config.UnmanagedClusterConfig) (*cluster.Kuberne
 
 	clusterManager := cluster.NewClusterManager(scConfig)
 
+	issues := clusterManager.PreflightCheck()
+	if issues != nil {
```

Comment:
In order to allow this, we'll need a way to let users override preflight checks.

`--skip-preflight` flag?

Reply:
Can you expand on what you're thinking here? Do we want to allow skipping?

Reply:
Yes, I don't want it to be entirely blocked at a code level with no way to bypass preflight checks

Reply:
Added